mpmath/ctx_mp_python.py

```diff
@@ -356,6 +356,64 @@ def __rpow__(s, t):
         return t ** s
 
     def __format__(s, format_spec):
+        """
+        ``mpf`` objects allow for formatting similar to Python floats:
+
+            >>> from mpmath import pi, mp
+            >>> mp.dps = 100
+            >>> print('{:105.100f}'.format(pi()))
+               3.1415926535897932384626433832795028841971693993751058209749445923078164062862089986280348253421170680
+            >>> print(f"{10*pi():.20e}")
+            3.14159265358979323846e+01
+
+        The format specification adopts the same general form as `Python's
+        Format Specification Mini-Language <FS_>`_. The only format types
+        supported by ``mpf`` are ``"e"``, ``"E"``, ``"f"``, ``"F"``, ``"g"`` ,
+        and ``"G"``.
```

Comment:
I would rather announce that *all* format types, valid for floating-point numbers in Python are supported.